corekit/src/main/java/com/midtrans/sdk/corekit/core/grouppayment/BankTransferCharge.java

```diff
@@ -2,13 +2,11 @@
 
 import com.midtrans.sdk.corekit.base.callback.MidtransCallback;
 import com.midtrans.sdk.corekit.core.snap.model.pay.request.CustomerDetailPayRequest;
-import com.midtrans.sdk.corekit.core.snap.model.pay.response.va.BcaPaymentResponse;
-import com.midtrans.sdk.corekit.core.snap.model.pay.response.va.BniPaymentResponse;
+import com.midtrans.sdk.corekit.core.snap.model.pay.response.BasePaymentResponse;
 import com.midtrans.sdk.corekit.core.snap.model.pay.response.va.OtherPaymentResponse;
-import com.midtrans.sdk.corekit.core.snap.model.pay.response.va.PermataPaymentResponse;
 import com.midtrans.sdk.corekit.utilities.Validation;
 
-public class MidtransBankTransfer extends BaseMidtransGroupPayments {
+public class BankTransferCharge extends BaseGroupPayments {
```

Comment:
Can we change `BaseGroupPayments` into `PaymentGroupBase` @suryagsaputra ?

Reply:
Done